webserver/webserver/src/main/java/io/helidon/webserver/HttpInitializer.java

```diff
@@ -131,6 +133,23 @@ void queuesShutdown() {
         });
     }
 
+    void updateSslContext(String socketName, SslContext context) {
+        if (sslContext == null) {
+            if (context != null) {
+                throw new IllegalStateException("TLS was not configured on the socket " + socketName + " before. Could not be "
```

Comment:
This sentence is a bit weird. Maybe just "Disabling TLS context at runtime is not allowed". Disable implies that it is currently enabled, I don't think we need to mention the socket name.

Reply:
Reworked